gravitational_teleport_rules/teleport_suspicious_commands.yml

```diff
@@ -0,0 +1,71 @@
+AnalysisType: rule
+Filename: teleport_suspicious_commands.py
+RuleID: Teleport.SuspiciousCommands
+DisplayName: Teleport Suspicious Commands Executed
+Enabled: true
+LogTypes:
+  - Gravitational.TeleportAudit
+Tags:
+  - SSH
+Severity: Medium
+Description: A user has invoked a suspicious command that could lead to a host compromise
+DedupPeriodMinutes: 60
+Reports:
+  MITRE ATT&CK:
+    - Execution:Command and Scripting Interpreter
+Reference: https://gravitational.com/teleport/docs/admin-guide/
+Runbook: >
+  Find related commands within the time window and determine if the command was invoked legitimately. Examine the arguments to determine how the command was used.
```

Comment:
I might also suggest reaching out to the user directly if the commands were suspicious.